diesel/src/query_dsl/save_changes_dsl.rs

```diff
@@ -114,13 +115,13 @@ where
 /// #     Ok(())
 /// # }
 /// ```
-pub trait SaveChangesDsl<Conn> {
+pub trait SaveChangesDsl<Conn>: Sized {
```

Comment:
The `Sized` bound should be on the method, not the trait. Putting it here is technically a breaking change.